priv/perf/apps/load_test/lib/child_chain/exit.ex

```diff
@@ -6,80 +20,135 @@ defmodule LoadTest.ChildChain.Exit do
   require Logger
 
   alias ExPlasma.Encoding
+  alias ExPlasma.Utxo
   alias LoadTest.ChildChain.Transaction
   alias LoadTest.Ethereum
+  alias LoadTest.Ethereum.Account
   alias LoadTest.Ethereum.Crypto
 
-  @gas_start_exit 400_000
+  @gas_start_exit 500_000
   @gas_challenge_exit 300_000
   @gas_add_exit_queue 800_000
   @standard_exit_bond 14_000_000_000_000_000
-  @token <<0::160>>
-  @vault_id 1
-
-  def start_exit(utxo_pos, tx_bytes, proof, from) do
-    opts =
-      tx_defaults()
-      |> Keyword.put(:gas, @gas_start_exit)
-      |> Keyword.put(:value, @standard_exit_bond)
-
-    {:ok, transaction_hash} =
-      Ethereum.contract_transact(
-        from,
-        contract_address_payment_exit_game(),
+  @poll_interval 1_000
+
+  @doc """
+  Returns the exit data of a utxo.
+  """
+  @spec get_exit_data(Utxo.t()) :: any()
+  def get_exit_data(%Utxo{} = utxo), do: get_exit_data(Utxo.pos(utxo))
+
+  @spec get_exit_data(non_neg_integer()) :: any()
+  def get_exit_data(utxo_pos) do
+    body = %WatcherSecurityCriticalAPI.Model.UtxoPositionBodySchema{
+      utxo_pos: utxo_pos
+    }
+
+    {:ok, response} =
+      WatcherSecurityCriticalAPI.Api.UTXO.utxo_get_exit_data(
+        LoadTest.Connection.WatcherSecurity.client(),
+        body
+      )
+
+    data = Jason.decode!(response.body)["data"]
+
+    %{
+      proof: data["proof"],
+      txbytes: data["txbytes"],
+      utxo_pos: data["utxo_pos"]
+    }
+  end
+
+  @doc """
+  Retries until the exit data of a utxo is found.
+  """
+  @spec wait_for_exit_data(Utxo.t(), pos_integer()) :: any()
+  def wait_for_exit_data(utxo_pos, counter \\ 10)
+  def wait_for_exit_data(_, 0), do: :error
+
+  def wait_for_exit_data(utxo_pos, counter) do
+    data = get_exit_data(utxo_pos)
+
+    if data.proof do
+      data
+    else
+      _ = Logger.info("Waiting for exit data")
+      Process.sleep(@poll_interval)
+      wait_for_exit_data(utxo_pos, counter - 1)
+    end
```

Comment:
`case` statements are preferred to avoid ambiguity, use if/else for trailing question mark function only